### PR TITLE
Adding country hero images

### LINKED ISSUE #125: Hero images for countries
### The feature

Add hero images to the country pages

### Why?

This would make the country pages less sterile and give a flavor of each country.

### Examples and inspiration

This could be solved by using [page banner (P948)](https://www.wikidata.org/wiki/Property:P948) which is a property all countries already have set. WE could try using it as is or with a grayscale and blue multiply in CSS.

### PR DESCRIPTION
**Pull request by Ainali**
Getting images through Wikivoyage.

I am not completely happy with the CSS in the header on each country page, but I couldn't figure out another way to do this.

Also, indentation is not my strong game, I guess this was a bit ugly.

Fixes #125

### CODE REVIEW

**Comment by Ainali** [comment]
This is still in draft until we figure out a good way to credit the images.

**Comment by Ainali** [comment]
Yeah, Wikimedia isn't really role models with regard to Creative Commons licenses since they have the attitude that an unnamed link from an image to the license information (on the file page on Wikimedia Commons) is enough. I mean technically it probably holds, but we should do better. Just adding a link on the image with the link text `license information` that links to file page would be better in my opinion and perhaps what we should go with?

**Comment by Ainali** [comment]
Here is a draft for a query to get information to give good credits: https://w.wiki/4LiY
Thanks to @lucaswerkmeister!

File: queries/countries.rq [hunk 1]
```diff
@@ -4,6 +4,10 @@ SELECT
   ?safeName
   ?description
   (SAMPLE(?website) AS ?website)
+  (SAMPLE(?banner) AS ?banner)
```
Comment (Abbe98) [inline queries/countries.rq:7]: Note to self: we need to make this sample earlier to ensure it matches the variables we get from Commons.

**Comment by Abbe98** [comment]
I ended up getting slightly stuck on the attribution part, I wanted to use the non-HTML representation for `?artist` because it can contain a ton of different HTML which I don't think we should consider safe, there does however not exist such a non-HTML representation.

I think the perfect attribution HTML would look like:

```html
<a href="#commons-description-page">LICENSE by author 1, author 2, author 3</a>
```

The page link and License are simple but I'm unable to figure out a good way to deal with the authors.

**Comment by Ainali** [comment]
If the HTML is not safe, then there is a problem at Wikimedia Commons as well. Sure, it is risk, but I think that it is largely negligible.

**Comment by Abbe98** [comment]
The big issues with the HTML are two; "unlimited" styling(which on WMF projects is a community matter), and random links. I'm unable to get these 8 attributions remotely consistent.

**Comment by Ainali** [comment]
This makes me wonder, on the Commons pages in the top there is an alternative for "Use this file on the web" which even for the [Ghana image](https://commons.wikimedia.org/wiki/File:Ghana_Cape_Coast_banner1.jpg) with a list in the authors field manages to produce this html:

```html
<a href="https://commons.wikimedia.org/wiki/File:Ghana_Cape_Coast_banner1.jpg">Ghana_Cape_Coast_01.jpg: Erik Cleves Kristensenderivative work: Jjtkk</a>, <a href="https://creativecommons.org/licenses/by/2.0">CC BY 2.0</a>, via Wikimedia Commons
```

Is it possible to reuse that logic?

**Comment by Abbe98** [comment]
It seems to use the same API calls as we do but with additional cleaning of the data:

https://commons.wikimedia.org/wiki/Commons:Machine-readable_data#MediaWiki_API & https://commons.wikimedia.org/wiki/MediaWiki:Gadget-Stockphoto.js#L-98

Maybe we could do something like the cleaning using SPARQL's `REPLACE`.

**Comment by Ainali** [comment]
Closing this PR now. Let's pick it up again when Commons is better at providing attributions.

**Pull request closed without merging**